Solopov/DZ5_AjaxAsynchronousMethodAndCSS/ServletTestList/src/main/java/TestClass.java

```diff
@@ -0,0 +1,20 @@
+import hibernate.entity.Question;
+import hibernate.entity.Quiz;
+import hibernate.service.ServiceQuestion;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.List;
+
+public class TestClass {
+
+    ServiceQuestion quizQuestion=new ServiceQuestion();
```

Comment:
This should not be in commit.

Reply:
Kkhm,sorry.. this class can't be there. It was create for testing functional of my hibernate methods.
By the way, I tried to create beans, but specifically in this class they did not initialize. Unfortunately, the reason for this was not found

Reply:
How you try to create beans here?

Reply:
I was used Autowired annotation over the composition field ServiceQuestion;